asymmetricrypt/asymmetricrypt/2017-11-20.yaml

```diff
@@ -0,0 +1,8 @@
+title:      Padding Oracle Vulnerability in RSA Encryption
+link:       https://github.com/Cosmicist/AsymmetriCrypt/issues/4
+cve:        ~
+branches:
+    0.x:
+        time:     2017-01-19 13:03:00
```

Comment:
Time is the time when the package was fixed. Not sure what to do here as there is no fix AFAIU. We need to have a new convention. I would suggest to keep the time empty.

Reply:
Propably `~` should be used. 
I just saw it a few times for missing cve-identifiers (like here).

Reply:
Okay, I'll send a superseding commit right now, then.

Reply:
Looks like it's failing because `time` is required. Hmm.

Reply:
Indeed, it's not supported yet by the validator and the server. So, I will need to make some adjustments before merging this PR.

Reply:
What if we relax that constraint but only if another key is set (e.g. `unfixed: TRUE`)